### PR TITLE
Move InAttribute from S.R.InteropServices to System.Runtime next to OutAttribute

### PR DESCRIPTION
**Pull request by ahsonkhan**
cc @VSadov, @jkotas

### CODE REVIEW

File: src/System.Runtime.InteropServices/ref/System.Runtime.InteropServices.cs [hunk 1]
```diff
@@ -7,11 +7,14 @@
 
 // Types moved down into System.Runtime.Handles
```
Comment (jkotas) [inline src/System.Runtime.InteropServices/ref/System.Runtime.InteropServices.cs]: Nit: This comment is not correct. All these types are in System.Runtime

File: src/System.Runtime/ref/System.Runtime.cs [hunk 2]
```diff
@@ -323,7 +323,7 @@ public abstract partial class Array : System.Collections.ICollection, System.Col
         int System.Collections.IStructuralEquatable.GetHashCode(System.Collections.IEqualityComparer comparer) { throw null; }
         public static bool TrueForAll<T>(T[] array, System.Predicate<T> match) { throw null; }
     }
-    [System.Runtime.InteropServices.StructLayoutAttribute(System.Runtime.InteropServices.LayoutKind.Sequential)]
+    [System.Runtime.InteropServices.StructLayoutAttribute(System.Runtime.InteropServices.LayoutKind.Sequential)]
```
Comment (jkotas) [inline src/System.Runtime/ref/System.Runtime.cs]: Nit: Extra white space

File: src/System.Runtime.InteropServices/ref/System.Runtime.InteropServices.cs (hunk 1 above)
Comment (ahsonkhan) [reply src/System.Runtime.InteropServices/ref/System.Runtime.InteropServices.cs]: I see. Double hop to System.Runtime it looks like: https://github.com/dotnet/corefx/blob/master/src/System.Runtime.Handles/ref/System.Runtime.Handles.Forwards.cs#L9

**Comment by ahsonkhan** [comment]
@dotnet-bot test NETFX x86 Release Build
@dotnet-bot test Linux x64 Release Build

**Pull request merged**